<launch>
    <!-- cuando se cree el objeto del controller mandarle los Kpid desde este launch cmo params -->
    <!-- publicar las max velocidades a un topico -->
    <!-- mover el goal a una velocidad constante -->
    <!-- tire warning de que los thrusters estan apagados -->
    <!-- <node name="pid_controller" pkg="girona_utils" type="pid_action" output="screen">
        <param name="max_vel" value="0.3"/>
        <param name="max_rot_vel" value="0.35"/>
        <param name="output_interface" value="cola2_controller"/>
    </node> -->

    <arg name="robot" default="girona500" doc="wich robot are you using?"/>
    <node name="pursuit_controller" pkg="girona_utils" type="pursuit_controller" output="screen">
        <param name="max_vel" value="0.3"/>
        <param name="max_rot_vel" value="0.35"/>
        <param name="output_interface" value="cola2_controller"/>
        <param name="base_link" value="$(arg robot)/base_link"/>
        <param name="velocity_topic" value="/$(arg robot)/controller/body_velocity_req"/>
    </node>

</launch>